website/blog/2024-08-29-poll-joining.md

```diff
@@ -45,4 +45,4 @@ We removed the entire message accumulator data structure that had exponential sp
 We are not stopping here, more interesting protocol upgrades are on the way that would improve the security of the protocol even further and eliminate another significant concern. Be patient, and enjoy the privacy!
 
 ### Who are we?
-Visit 3327.io and find out.
\ No newline at end of file
+Visit 3327.io and find out. Special kudos to our [Mihailo Radojević](https://github.com/radojevicMihailo) and [Boris Cvitak](https://github.com/djanluka) who did a lot of heavy lifting so these cool features could be here today!
```

Comment:
Please add the blog as a separate PR to the dev branch, so that it can be released sooner

Reply:
https://github.com/privacy-scaling-explorations/maci/pull/1798